### ISSUE TITLE
Persistent crash when opening certain SoundCloud URLs

### ISSUE THREAD

**Issue by Shizmob** [TeamNewPipe/NewPipe#2450]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [x] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

Thanks for developing this app, I use it everyday to great pleasure! Since a month or so opening a wide variety of SoundCloud URLs, but not all, causes the application to just crash into a bugreport screen. An example of an affected URL is https://soundcloud.com/psyqui/dj-noriken-feat-yuce-psyqui-remix.

## Exception
* __User Action:__ requested stream
* __Request:__ https://soundcloud.com/psyqui/dj-noriken-feat-yuce-psyqui-remix
* __Content Language:__ GB
* __Service:__ SoundCloud
* __Version:__ 0.16.2
* __OS:__ Linux Android 9 - 28

<details><summary><b>Crash log</b></summary><p>

```
org.schabi.newpipe.extractor.exceptions.ParsingException: Attempt to invoke virtual method 'java.lang.String org.jsoup.nodes.Element.attr(java.lang.String)' on a null object reference
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudStreamLinkHandlerFactory.getUrl(SoundcloudStreamLinkHandlerFactory.java:25)
	at org.schabi.newpipe.extractor.linkhandler.LinkHandlerFactory.fromUrl(LinkHandlerFactory.java:47)
	at org.schabi.newpipe.extractor.StreamingService.getStreamExtractor(StreamingService.java:288)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:59)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:115)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$5fJcha6Sq5APJBLdG6osaJby-mc.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14479)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.jsoup.nodes.Element.attr(java.lang.String)' on a null object reference
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudParsingHelper.resolveUrlWithEmbedPlayer(SoundcloudParsingHelper.java:104)
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudStreamLinkHandlerFactory.getUrl(SoundcloudStreamLinkHandlerFactory.java:23)
	... 29 more

```
</p></details>
<hr>

**Comment by theScrabi** [comment]
Confimed

**Comment by Stypox** [comment]
This was already reported in #2362. Too bad the issue also contains 8 other problems, so it's uncomfortable and not searchable.

**Comment by jimbo1qaz** [comment]
I noticed this weeks ago, saw no tickets here, but filed a crash report via email. Are emailed crash reports ignored or not paid attention to?

Are crash reports allowed or banned in the issue tracker? Why does https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md#crash-reporting say "Do not report crashes in the GitHub issue tracker.", but the issue tracker template mentions the "incredible bugreport to markdown converter" (for JSON crash reports)?

**Comment by gkeegan** [comment]
@jimbo1qaz

You should use the converter since it basically just formats the JSON so that it is easy to read on GitHub.

Submitting crash reports isn't banned afaik, and are pretty useful when just one is posted, but often when things start crashing it turns into 100 people posting the same issue which is a nightmare for keeping the tracker clean.

About the emails, it might be that @tobigr might not have permissions to see them? I'll leave that part up to him to answer though.

**Comment by Stypox** [comment]
This was a problem with SoundCloud extractor and was fixed in TeamNewPipe/NewPipeExtractor#2344
